ksmt-core/src/main/kotlin/org/ksmt/expr/KFloatingPointExpr.kt

```diff
@@ -416,6 +463,10 @@ class KFpRemExpr<S : KFpSort> internal constructor(
     }
 
     override fun accept(transformer: KTransformerBase): KExpr<S> = transformer.transform(this)
+
+    override fun internHashCode(): Int = hash(arg0, arg1)
+    override fun internEquals(other: Any): Boolean =
+        structurallyEqual(other, { arg0 }, { arg1 })
```

Comment:
Maybe we could create overloads for values without lambdas?

Reply:
AFAIK, it requires reflection.